core/examples/disable_ops.rs

```diff
@@ -9,7 +9,7 @@ use deno_core::RuntimeOptions;
 fn main() {
   let my_ext = Extension::builder()
     .middleware(|name, opfn| match name {
-      "op_print" => deno_core::void_op_sync(),
+      "op_print" => deno_core::void_op_sync::v8_cb(),
```

Comment:
What's the difference between `v8_cb` and `decl`?

Reply:
`decl` conceptually is all of the revelant metadata required for declaring an op, right now that's a name and a function pointer. But thay may evolve. `v8_cb` returns the function pointer component of `decl()`.

The idea being that we want the op to preserve it's original name but ultimately call a different implementation.

Happy to tweak our middleware impl so middlewares return `::decl()` and handles the name preservation

Reply:
Will do in a follow-up